### ISSUE TITLE
EOL: Look at VIVO errors that are being reported by a particular ontology

### ISSUE THREAD

**Issue by daniels303** [NCAR/2014-EarthCube-BuildingBlocks-EnablingCollaboration-14402930#12]

**Comment by daniels303** [comment]
Assign to John.

**Comment by daniels303** [comment]
Reports "an error occurs during reasoning"

**Comment by daniels303** [comment]
Based on the talk with Benjamin and the ontologies their instance uses, it looks like the ontology causing this might be GCIS or ISO.

**Comment by jja** [comment]
````
2017-04-11 16:25:24,407 WARN  [OntDocumentManager] An error occurred while attempting to read from http://www.geosciml.org/vocabularies/iso-19115-codelists.owl. Msg was 'null'.
java.lang.NullPointerException
        at org.apache.jena.riot.web.HttpOp.execHttpGet(HttpOp.java:172)
        at org.apache.jena.riot.stream.LocatorURL.open(LocatorURL.java:47)
        at org.apache.jena.riot.stream.StreamManager.openNoMapOrNull(StreamManager.java:124)
        at org.apache.jena.riot.stream.StreamManager.open(StreamManager.java:81)
        at org.apache.jena.riot.RDFDataMgr.open(RDFDataMgr.java:725)
        at org.apache.jena.riot.RDFDataMgr.open(RDFDataMgr.java:714)
        at org.apache.jena.riot.RDFDataMgr.parse(RDFDataMgr.java:649)
        at org.apache.jena.riot.RDFDataMgr.read(RDFDataMgr.java:211)
        at org.apache.jena.riot.RDFDataMgr.read(RDFDataMgr.java:184)
        at org.apache.jena.riot.RDFDataMgr.read(RDFDataMgr.java:122)
        at org.apache.jena.riot.RDFDataMgr.read(RDFDataMgr.java:113)
        at org.apache.jena.riot.adapters.RDFReaderRIOT.read(RDFReaderRIOT.java:77)
        at com.hp.hpl.jena.rdf.model.impl.ModelCom.read(ModelCom.java:247)
        at com.hp.hpl.jena.util.FileManager.readModelWorker(FileManager.java:378)
        at com.hp.hpl.jena.util.FileManager.readModel(FileManager.java:345)
        at com.hp.hpl.jena.util.FileManager.readModel(FileManager.java:329)
        at com.hp.hpl.jena.ontology.OntDocumentManager.read(OntDocumentManager.java:1061)
        at com.hp.hpl.jena.ontology.OntDocumentManager$1.readModel(OntDocumentManager.java:1031)
        at com.hp.hpl.jena.rdf.model.impl.ModelMakerImpl.getModel(ModelMakerImpl.java:104)
        at com.hp.hpl.jena.ontology.OntDocumentManager.fetchLoadedImportModel(OntDocumentManager.java:1027)
        at com.hp.hpl.jena.ontology.OntDocumentManager.fetchPossiblyCachedImportModel(OntDocumentManager.java:1000)
        at com.hp.hpl.jena.ontology.OntDocumentManager.loadImport(OntDocumentManager.java:973)
        at com.hp.hpl.jena.ontology.OntDocumentManager.loadImports(OntDocumentManager.java:767)
        at com.hp.hpl.jena.ontology.OntDocumentManager.loadImports(OntDocumentManager.java:705)
        at com.hp.hpl.jena.ontology.impl.OntModelImpl.loadImports(OntModelImpl.java:1980)
        at com.hp.hpl.jena.ontology.impl.OntModelImpl.read(OntModelImpl.java:2250)
        at edu.cornell.mannlib.vitro.webapp.servlet.setup.FileGraphSetup.readGraphs(FileGraphSetup.java:166)
        at edu.cornell.mannlib.vitro.webapp.servlet.setup.FileGraphSetup.contextInitialized(FileGraphSetup.java:94)
        at edu.cornell.mannlib.vitro.webapp.startup.StartupManager.initialize(StartupManager.java:179)
        at edu.cornell.mannlib.vitro.webapp.startup.StartupManager.contextInitialized(StartupManager.java:65)
````

**Comment by dwstott** [comment]
link working now
	http://www.geosciml.org/vocabularies/iso-19115-codelists.owl <http://www.geosciml.org/vocabularies/iso-19115-codelists.owl>

**Comment by jja** [comment]
> link working now

As always. I don't think (y)our ability to download via browser has anything to do with the error.

**Comment by jja** [comment]
adding the owl file to vivo src does not help, it still tries & fails to download (I'm pretty sure I tried this long ago and that's where I've left it...)

**Comment by jja** [comment]
bug in jena fixed in 2.11.1, need to update Vitro/VIVO to fully use ISO schemata ( #5 )

**Comment by jja** [comment]
VIVO/Vitro 1.9.3 is still using Jena 2.10.1 so the ISO codelists may not be usable.
